services/core-api/app/api/variances/models/variance.py

```diff
@@ -42,6 +47,11 @@ class Variance(SoftDeleteMixin, AuditMixin, Base):
     received_date = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
     expiry_date = db.Column(db.DateTime)
 
+    created_by = db.Column(db.String, default=User().get_user_username)
```

Comment:
these column definitions shouldn't be needed here since this model inherits the audit mixin